### ISSUE TITLE
Professor see all student and can talk to one by one for exam

### ISSUE THREAD

**Issue by Entretoize** [bigbluebutton/bigbluebutton#15483]
Actually I can create a room where students can see their professor that will be the presenter, but a student can't talk to the professor (except by chat) without all students see and ear him. This is annoying for exam.

It would be perfect to have a mode where the presenter see all cams but users see only the presenter and only if he is talking the them. And the presenter will be able to choose to who he is talking.

I though about creating a room for each user and using iframe to display all students to the professor, but I'm not sure how to handle private conversation, I have to turn off camera and microphone to all non active iframes, I'm not sure it will work.

**Comment by hiroshisuga** [comment]
Agreed. But this issue has been raised a couple of times already, for instance #12199.

**Comment by Entretoize** [comment]
It is similar but not exactly the same, breakout rooms are unpractical (for me).

**Comment by ffdixon** [comment]
We want to give the teacher the ability to talk one-on-one with a student, which we internally call "cone of silence".

**Comment by Entretoize** [comment]
I finally create one room per student, and the teacher choose the one he wants...

**Comment by ffdixon** [comment]
That works as well.  We're working towards giving you the ability to do that within BigBlueButton itself.
